### PR TITLE
stat/distuv: Add documentation to `interfaces.go`

### PR DESCRIPTION
**Pull request by romanwerpachowski**
Please take a look.

### CODE REVIEW

File: stat/distuv/interfaces.go [hunk 1]
```diff
@@ -4,19 +4,23 @@
 
 package distuv
 
+// LogProber calculates the log-probability for a given value of a random variable.
```
Comment (kortschak) [inline stat/distuv/interfaces.go:7]:
```
// LogProber wraps the LogProb method.
```

File: stat/distuv/interfaces.go [hunk 2]
```diff
@@ -4,19 +4,23 @@
...
 type LogProber interface {
 	LogProb(float64) float64
 }
 
+// Rander samples a random variable.
```
Comment (kortschak) [inline stat/distuv/interfaces.go:12]:
```
// Rander wraps the Rand method.
```

File: stat/distuv/interfaces.go [hunk 3]
```diff
@@ -4,19 +4,23 @@
...
 type Rander interface {
 	Rand() float64
 }
 
+// RandLogProber is a Rander and a LogProber.
```
Comment (kortschak) [inline stat/distuv/interfaces.go:17]:
```
// RandLogProber is the interface that groups the Rander and LogProber methods.
```

File: stat/distuv/interfaces.go [hunk 4]
```diff
@@ -4,19 +4,23 @@
...
 type RandLogProber interface {
 	Rander
 	LogProber
 }
 
+// Quantiler calculates the quantile of a distribution for given probability.
```
Comment (kortschak) [inline stat/distuv/interfaces.go:23]:
```
// Quantiler wraps the Quantile method.
```

File: stat/distuv/interfaces.go [hunk 5]
```diff
@@ -4,19 +4,23 @@
 
 package distuv
 
+// LogProber calculates the log-probability for a given value of a random variable.
 type LogProber interface {
 	LogProb(float64) float64
```
Comment (kortschak) [inline stat/distuv/interfaces.go:9]:
```
// LogProb returns the natural logarithm of the
// value of the probability density function at x.
LogProb(x float64) float64
```

File: stat/distuv/interfaces.go [hunk 6]
```diff
@@ -4,19 +4,23 @@
...
 }
 
+// Rander samples a random variable.
 type Rander interface {
 	Rand() float64
```
Comment (kortschak) [inline stat/distuv/interfaces.go:14]:
```
// Rand returns a random sample drawn from the distribution.
Rand() float64
```

File: stat/distuv/interfaces.go [hunk 7]
```diff
@@ -4,19 +4,23 @@
...
 
+// RandLogProber is a Rander and a LogProber.
 type RandLogProber interface {
 	Rander
 	LogProber
 }
 
+// Quantiler calculates the quantile of a distribution for given probability.
 type Quantiler interface {
 	Quantile(p float64) float64
```
Comment (kortschak) [inline stat/distuv/interfaces.go:25]:
```
// Quantile returns inverse of the cumulative distribution
// function for the probability p.
```

Comment (romanwerpachowski) [reply stat/distuv/interfaces.go:25]: I'll use the new wording which we agreed for Bernoulli.

File: stat/distuv/interfaces.go (hunk 5 above)
Comment (romanwerpachowski) [reply stat/distuv/interfaces.go:9]: What about discrete distributions, which do not have the PDF but a probability mass function?

**Comment by romanwerpachowski** [comment]
PTAL. I left some questions.

File: stat/distuv/interfaces.go (hunk 7 above)
Comment (kortschak) [reply stat/distuv/interfaces.go:27]: Can you wrap the text more tightly; godoc will format text in non-code chunk doc comments, but does not do this for comments in struct or interface definitions, so we need to be more careful there. Something on the order of 50 characters is pretty good.

File: stat/distuv/interfaces.go (hunk 5 above)
Comment (kortschak) [reply stat/distuv/interfaces.go:9]:
```
// LogProb returns the natural logarithm of the
// value of the probability density or probability
// mass function at x.
LogProb(x float64) float64
```
?

Presumably equivalent text in `Prober`.

Comment (romanwerpachowski) [reply stat/distuv/interfaces.go:9]: Done. Where's `Prober`?

File: stat/distuv/interfaces.go (hunk 7 above)
Comment (romanwerpachowski) [reply stat/distuv/interfaces.go:27]: Done.

**Comment by romanwerpachowski** [comment]
PTAL.

File: stat/distuv/interfaces.go (hunk 5 above)
Comment (kortschak) [reply stat/distuv/interfaces.go]: Doesn't exist yet I suppose. We don't need it if it's not there.

**Pull request merged**